File: SW Expert Academy/D1/2068 최대수 구하기.py
'''
 # SW Expert Academy 2068: 최대수 구하기
 # D1
 # 21.1.31 try: 1
'''
#sys.stdin = open("input.txt", "r") #import sys

T = int(input())
for test_case in range(1, T + 1):
    l = [int(x) for x in input().split()]

    print(f"#{test_case} {max(l)}")

File: SW Expert Academy/D1/2068 최대수 구하기.java
/*******************************************
 * SW Expert Academy 2068: 최대수 구하기
 * D1
 * 21.1.31 try: 1
 *******************************************/
import java.util.Scanner;

class Solution {
    public static void main(String args[]) throws Exception {
        //System.setIn(new FileInputStream("res/input.txt"));

        Scanner sc = new Scanner(System.in);
        int T;
        T=sc.nextInt();

        for(int test_case = 1; test_case <= T; test_case++) {
            int num, max = 0;

            for (int i = 0; i < 10; ++i) {
                num = sc.nextInt();

                if (num > max) max = num;
            }

            System.out.println("#" + test_case + ' ' + max);
        }

        sc.close();
    }
}